packages/app/background-jobs-common/src/background-job-processor/processors/FakeBackgroundJobProcessor.ts

```diff
@@ -41,22 +37,7 @@ export class FakeBackgroundJobProcessor<
       },
     )
   }
-  protected override process(job: Job<JobData>): Promise<void> {
-    this._processCalls.push(job.data)
+  protected override process(): Promise<void> {
     return Promise.resolve()
   }
-
-  /**
-   * @deprecated use job spy instead
-   */
-  public get processCalls(): JobData[] {
-    return this._processCalls
-  }
```

Comment:
Using major version to remove this, long time since spies were implemented, we should be in a good place to remove it already